build.sbt

```diff
@@ -128,8 +128,8 @@ lazy val testing = crossProject(JSPlatform, JVMPlatform)
   .settings(
     name := "log4cats-testing",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-effect"     % catsEffectV,
```

Comment:
Have no idea how updating the CE version caused this formatting change 🙄

Reply:
Yikes, if you merge this it'll conflict with https://github.com/typelevel/log4cats/pull/592 😬

Reply:
No worry, I fixed it.